index.js

```diff
@@ -169,7 +171,9 @@ module.exports = {
         disablePolyfill: false,
         autoPolyfill: false,
         inputPath: 'translations',
-        outputPath: 'translations'
+        outputPath: 'translations',
+        failOnMissingTranslation: false,
```

Comment:
`allowMissingTranslation() { return false; }` could be the same as `failOnMissingTranslation: true` - correct?

Reply:
Any suggestions on how to improve this?
Right now the missing translation predicate also causes log to not emit missing translations if it returns true.

Reply:
Let me give it some thought and get back!

Reply:
Thoughts on combining these two into a single method `onMissingTranslation` where by default it will log but if the user implements this hook, they'll be able to `throw` or conditionally log in the case they want to fail on missing translations.

That should solve both of the issues, let me know if you disagree!

Reply:
I'm not sure if it's a good idea to let the user build the error to throw.

I'm also not sure how this will work if a user wants to throw all missing translations. If the method is called for each missing translation then a user has to collect all missing translations and throw in the end.

Maybe we could rename the options to make it more clear?
e.g. `requiresTranslation(key, languages)` and `throwMissingTranslations`.

Reply:
@makepanic good feedback.  I'm on board with the renaming.